lib/gateway/Shard.js

```diff
@@ -603,20 +609,26 @@ class Shard extends EventEmitter {
                             reactionObj.me = false;
                         }
                     }
+                } else {
+                    message = {
+                        id: packet.d.message_id,
+                        channel: channel || {id: packet.d.channel_id}
+                    };
+
+                    if(packet.d.guild_id) {
+                        message.guildID = packet.d.guild_id;
+                    }
                 }
                 /**
                 * Fired when someone removes a reaction from a message
                 * @event Client#messageReactionRemove
-                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. If the channel is not cached, channel key will be an object with only an id. No other property is guaranteed
+                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel`. If the channel is not cached, channel key will be an object with only an id. `guildID` will be present if the message was sent in a guild channel. No other property is guaranteed
```

Comment:
```suggestion
                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. If the channel is not cached, channel key will be an object with only an id. `guildID` will be present if the message was sent in a guild channel. No other property is guaranteed
```
I'm guessing you accidentally removed this